smoke-tests/play/build.gradle

```diff
@@ -40,5 +40,7 @@ def targetJDK = project.hasProperty("targetJDK") ? project.targetJDK : 11
 jib {
   from.image = "bellsoft/liberica-openjdk-alpine:$targetJDK"
   to.image = "open-telemetry-docker-dev.bintray.io/java/smoke-play-jdk$targetJDK"
+  to.tags = [new Date().format("yyyyMMdd.HHmmSS")]
+  container.creationTime = 'USE_CURRENT_TIMESTAMP'
```

Comment:
Please use consistent order of configuration

Reply:
All `container` blocks are moved after `to`/`from` settings.